### PR TITLE
RWA-532: Multicategories support

### PR DESCRIPTION
**Pull request by alexbaeza**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-532
https://tools.hmcts.net/jira/browse/RWA-310

### Change description ###

Add camunda feign configuration.
Add ignore unknown to generic dmnValue pojo.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/service/CamundaClient.java [hunk 1]
```diff
@@ -8,24 +8,32 @@
...
 )
 public interface CamundaClient {
 
     String SERVICE_AUTHORIZATION = "ServiceAuthorization";
 
-    @PostMapping(value = "/message", produces = MediaType.APPLICATION_JSON_VALUE)
+    @PostMapping(
+        value = "/message",
+        consumes = MediaType.APPLICATION_JSON_VALUE,
+        produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (indvamshi) [inline src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/service/CamundaClient.java:29]: not required.

Comment (alexbaeza) [reply src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/service/CamundaClient.java:29]: Hi @indvamshi it was required to add the consume as by default the content type was:
x-www-form-urlencoded as you can see it now inherits from the CamundaFeignConfiguration.class

**Review by indvamshi — APPROVED** [review]

**Review by iguannago — APPROVED** [review]

**Review by BenNzewi — CHANGES_REQUESTED** [review]
EvaluateDMNTest -> should_evaluate_and_return_dmn_results fails due to dmn changes. The following line ".body("results[0].taskCategory.value", equalTo("Case progression"));" should no longer use taskCategory or value Case progression".   Should be ".body("results[0].processCategories.value", equalTo("caseProgression"));"

**Comment by alexbaeza** [comment]
> EvaluateDMNTest -> should_evaluate_and_return_dmn_results fails due to dmn changes. The following line ".body("results[0].taskCategory.value", equalTo("Case progression"));" should no longer use taskCategory or value Case progression". Should be ".body("results[0].processCategories.value", equalTo("caseProgression"));"

Good spot I raised another or to update this test after the dmn has been updated:
https://github.com/hmcts/wa-workflow-api/pull/237

**Review by BenNzewi — APPROVED** [review]

**Pull request merged**